packages/api-fetch/src/index.ts

```diff
@@ -114,11 +114,23 @@ const defaultFetchHandler: FetchHandler = ( nextOptions ) => {
 				throw err;
 			}
 
-			// Otherwise, there is most likely no network connection.
-			// Unfortunately the message might depend on the browser.
+			// If the browser reports being offline, we'll just assume that
+			// this is why the request failed.
+			if ( ! window.navigator.onLine ) {
+				throw {
+					code: 'offline_error',
+					message: __(
+						'Unable to connect. Please check your Internet connection.'
```

Comment:
I was unsure about the capitalization of "Internet" but I guess there's [data behind it](https://en.wikipedia.org/wiki/Capitalization_of_Internet), as close to a tie as that might be 😅 .

Reply:
On the other hand, Core uses lower-case "internet". Although the sample is very small, there are just two messages that use the word:
- "Keeping your site updated is important for security. It also makes the internet a safer place for you and your readers."
- "The internet address of your network will be %s." (when configuring the network feature)

Capital I is used only when talking about Internet Explorer.

Reply:
I'm not invested at all, was just curious about which convention is actually used in WordPress. 🙂 The Gutenberg repo also leans heavily towards lowercase, although it's all comments and docs, there's zero usage in code.